src/Transformer.hpp

```diff
@@ -260,74 +298,142 @@ namespace transformer
+         * @param name            name of the stream. This is only for debug
+         *                        purposes
+         * @result                stream index, which is used to identify the
+         *                        stream (e.g. for push).
          * */
         template <class T> int registerDataStreamWithTransform(base::Time dataPeriod, Transformation &transformation, boost::function<void (const base::Time &ts, const T &value, const Transformation &t)> callback, int priority = - 1, const std::string &name = std::string())
         {
             return aggregator.registerStream<T>(boost::bind( callback, _1, _2, boost::ref(transformation) ), 0, dataPeriod, priority, name);
         };
 
         /**
-         * This function unregistes a data stream. 
+         * This function will remove the stream with the given index from the
+         * stream aligner.
+         *
+         * @param idx index of the stream that should be unregistered
          * */
         void unregisterDataStream(int idx)
         {
             return aggregator.unregisterStream(idx);
         };
 
         /** 
-         * @copydoc aggregator::StreamAligner::disableStream
+         * Will disable the stream with the given index.
+         *
+         * All data left in the stream will still be played out, however the
+         * stream will be ignored for lookahead and timeout calculation. A
+         * stream, which is disabled can be enabled through the enableStream()
+         * call, or if new data in this stream arrives.
+         *
+         * The functionality is needed for cases where streams might be
+         * optional, so that the other streams won't be delayed up to the
+         * maximum time out value.
+         *
+         * @param idx index of the stream that should be disabled
          */
         void disableStream( int idx )
         {
             return aggregator.disableStream( idx );
         }
 
         /** 
-         * @copydoc aggregator::StreamAligner::enableStream
+         * Enables a stream which has been disabled previously.
+         *
+         * All streams are enabled by default. Does not have any effect on
+         * streams which are already enabled.
+         *
+         * @param idx index of the stream that should be enabled
          */
         void enableStream( int idx )
         {
             return aggregator.enableStream( idx );
         }
 
         /** 
-         * @copydoc aggregator::StreamAligner::isStreamActive
+         * See if a stream is enabled or disabled.
+         *
+         * @param idx index of the stream to look at
+         * @return true if the stream is enabled (active)
          */
         bool isStreamActive( int idx ) const 
         {
             return aggregator.isStreamActive( idx );
         }
         
+        /**
+         * @deprecated
+         *
+         * This never worked, except for boolean typed streams.
+         */
```

Comment:
From a git blame, this is meant to be used with a stream created with registerTransfromCallback to only get transformations and not data.

Reply:
It was introduced in a35ede8dd0f84dfefb36627033c1df5c72b56ff1, together with a registerTransfromCallback returning aggregator.registerStream<bool>(). That was removed/changed in the meantime, in 299857530298fd43912d341cdd273dc20c83fd9a, where the name spelling was fixed and it no longer uses that helper stream, also removing the need to call requestTransformationAtTime.

I'll remove the "never" in the comment and give some context.

Reply:
Got it. Thanks.